creator/studies/migrations/0031_patch_dataservice.py

```diff
@@ -0,0 +1,118 @@
+# Generated by Django 2.2.24 on 2021-11-02 18:23
+
+"""
+This migration patches the new attributes into the DataService Study
+entities using the spreadsheet found in this migrations folder.
+"""
+
+from django.conf import settings
+from django.db import migrations, models
+
+import pandas as pd
+import os
+import requests
+
+
+def patch_helper(kf_id, study_dict):
+    """
+    Helper function to patch the data in study_dict to the kf_id Study in
+    DataService.
+    """
+    # Patch DataService
+    try:
+        resp = requests.patch(
+            f"{settings.DATASERVICE_URL}/studies/{kf_id}",
+            json=study_dict,
+            timeout=settings.REQUESTS_TIMEOUT,
+            headers=settings.REQUESTS_HEADERS,
+        )
+    except requests.exceptions.RequestException as e:
+        raise Exception(f"Problem updating study {kf_id}: {e}")
+    
+    # Raise an error if it looks like study failed to update
+    if not resp.status_code == 200 or "results" not in resp.json():
```

Comment:
I think if a study wasn't found (status 404) in dataservice we probably don't want to raise an exception. Probably just want to log it and return